README.md

```diff
@@ -1 +1,45 @@
-﻿# Todo List App
+
+# AwesomeBooks
+
+This is a wesbite where you can add and remove books.
+
+![screenshot](Screenshottt.png)
+
+This project is my portfolio.
+
+## Built With
+
+- Javascript
+- HTML
+- CSS
+- Wabpack
+
+[Live Demo](https://github.com/zairdon20/Todo.git)
+
+## Getting Started
+
+### install
+
+- run npm start to view the project locally.
+
+To get the content of this project locally you need to run this command in your terminal:
+
+- git clone your https://github.com/zairdon20/Todo.git
+- cd project My-Portfolio
```

Comment:
- [ ] [OPTIONAL] It is ok if you use the readme file from an other project, just make sure to delete all the information from the previous project as it makes the project look unorganized and less professional.